satellite/satellitedb/migrate.go

```diff
@@ -60,22 +52,13 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 	return &migrate.Migration{
 		Table: "versions",
 		Steps: []*migrate.Step{
+
 			{
-				// some databases may have already this done, although the version may not match
 				DB:          db.db,
 				Description: "Initial setup",
-				Version:     0,
+				Version:     65,
```

Comment:
I think this needs to be a 66- you've included the `root_piece_id` column that was added in migration 66.

Reply:
Doh, never mind, I see that 66 is below.